### PR TITLE
[11] Add email sending after creating a new order.

### PR DESCRIPTION
**Pull request by iggysav**

### CODE REVIEW

**Review by olegbrest1 — APPROVED** [review]

File: app/controllers/checkouts_controller.rb [hunk 1]
```diff
@@ -170,28 +171,24 @@ def order_params
...
-    # Spree::Auth::Config[:registration_step] &&
+    Spree::Auth::Config[:registration_step] &&
       !already_registered?
   end
 
-  # https://trello.com/c/vNXjjb2C/3-remove-my-account-from-ui
   def already_registered?
-     # spree_current_user || guest_authenticated?
-    guest_authenticated?
+    spree_current_user || guest_authenticated?
```
Comment (fooooxmr) [inline app/controllers/checkouts_controller.rb:191]: Why this is uncommented?

File: app/mailers/application_mailer.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  # https://trello.com/c/NpB7SvoO/11-add-email-sending-after-creating-a-new-order
+  default from: "store@richdoor.by"
```
Comment (fooooxmr) [inline app/mailers/application_mailer.rb:3]: Please, move email to ENV variable.

File: app/mailers/spree/order_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      subject = build_subject(t('.subject'), resend)
+      mail(to: @order.email, bcc: bcc_address(@store), from: from_address(@store), subject: subject)
+    end
+
+
+    #https://trello.com/c/NpB7SvoO/11-add-email-sending-after-creating-a-new-order
+    def confirm_email_admin(order, resend = false)
+       @order = order
+       @store = @order.store
+       order_number=@order.number
```
Comment (fooooxmr) [inline app/mailers/spree/order_mailer.rb]: Please, add spaces around =

File: app/mailers/spree/order_mailer.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+       subject = build_subject(t('.subject'), resend)
+       admin_emails = Spree::RoleUser.where(role_id: Spree::Role.where(name: "admin").pluck(:id)).pluck(:user_id).map { |user_id| Spree::User.find(user_id).email }
+       email_customer=Spree::Order.where(number: order_number).pluck(:email)
```
Comment (fooooxmr) [inline app/mailers/spree/order_mailer.rb]: Spaces.

File: app/overrides/my_app/spree/order/remove_checkout_step.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# https://trello.com/c/YZ0eSj8S/10-implement-creating-an-order-on-request-call-page
+module MyApp
+  module Spree
+    module Order
+      module RemoveCheckoutStep
+        def self.prepended(base)
+          # base.remove_checkout_step :address
```
Comment (fooooxmr) [inline app/overrides/my_app/spree/order/remove_checkout_step.rb]: Remove comment, please.

File: app/overrides/spree/country.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# https://trello.com/c/YZ0eSj8S/10-implement-creating-an-order-on-request-call-page
+class Spree::Country < Spree::Base
+  def self.default
+    # find_by!(iso: Spree::Config.default_country_iso)
```
Comment (fooooxmr) [inline app/overrides/spree/country.rb]: Remove comment.

**Review by fooooxmr — APPROVED** [review]

File: app/overrides/spree/country.rb (hunk 6 above)
Comment (iggysav) [reply app/overrides/spree/country.rb]: Done

File: app/overrides/my_app/spree/order/remove_checkout_step.rb (hunk 5 above)
Comment (iggysav) [reply app/overrides/my_app/spree/order/remove_checkout_step.rb]: Done

File: app/mailers/spree/order_mailer.rb (hunk 4 above)
Comment (iggysav) [reply app/mailers/spree/order_mailer.rb]: Done

File: app/mailers/spree/order_mailer.rb (hunk 3 above)
Comment (iggysav) [reply app/mailers/spree/order_mailer.rb]: Done

**Pull request merged**